toolset/travis/travis_setup.sh

```diff
@@ -2,18 +2,6 @@
 
 export DEBIAN_FRONTEND=noninteractive
 
-sudo apt-get update
-sudo apt-get install python-dev
-sudo pip install pyopenssl --upgrade
-sudo pip install ndg-httpsclient --upgrade
-sudo pip install pyasn1 --upgrade
-sudo pip install colorama
-# Version 2.3 has a nice Counter() and other features
-# but it requires —-allow-external and -—allow-unverified
-sudo pip install progressbar
-echo "127.0.0.1 " `hostname` | sudo tee /etc/hosts
-echo "127.0.0.1 localhost" | sudo tee /etc/hosts
```

Comment:
Let's allow a full Travis-CI run and we'll make sure nothing obvious breaks from removing that from `/etc/hosts`